k8s/controller/controller.yaml

```diff
@@ -11,6 +12,8 @@ spec:
     spec:
       containers:
       - name: training-job-controller
-        image: helinwang/training-job-controller
+        image: registry.baidu.com/paddlepaddle/controller:helin
```

Comment:
Not a general image, just use this for test.

Reply:
Curious is this has the same content as `helinwang/training-job-controller`? (otherwise what has been added?)

Reply:
Yes, the same as `helinwang/training-job-controller`, the internal CPU cluster use the internal registry server will be faster then the Docker Hub.